Activity.fxml

```diff
@@ -0,0 +1,213 @@
+            <children>
+                <HBox prefHeight="100.0" prefWidth="200.0">
+               <children>
+                  <Label fx:id="durationTooltip" ellipsisString="" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" opacity="0.0" prefHeight="13.0" prefWidth="13.0" text="My Image">
+                     <HBox.margin>
+                        <Insets left="5.0" top="5.0" />
+                     </HBox.margin>
+                  </Label>
+               </children>
+            </HBox>
+            <HBox prefHeight="100.0" prefWidth="200.0">
+               <children>
+                      <TextField fx:id="quantity" onKeyPressed="#limitTextInput" onKeyReleased="#validateQuantity" opacity="0.0" promptText="Quantity" />
+                  <Label fx:id="quantityTooltip" ellipsisString="" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="13.0" prefWidth="13.0" text="My Image">
+                     <graphic>
+                        <ImageView fitHeight="13.0" fitWidth="13.0" opacity="0.0" preserveRatio="true" smooth="true">
+                           <image>
+                              <Image url="@../content/information.png" />
+                           </image>
+                        </ImageView>
+                     </graphic>
+                     <HBox.margin>
+                        <Insets left="5.0" top="5.0" />
+                     </HBox.margin>
+                  </Label>
+               </children>
+            </HBox> 
+            </children>
+        </HBox>
+      <VBox prefHeight="200.0" prefWidth="100.0" GridPane.columnIndex="1" GridPane.rowIndex="1">
+         <children>
+            <Label text="Which day will the activity start?" textFill="WHITE" />
+		        <DatePicker fx:id="date" editable="false" onAction="#validateDate" promptText="Date" />
+              <HBox>
+              	<children>
+      	            <Label fx:id="dateTooltip" ellipsisString="" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="13.0" prefWidth="13.0" text="My Image">
+      	               <graphic>
+      	                  <ImageView fitHeight="13.0" fitWidth="13.0" opacity="0.0" preserveRatio="true" smooth="true">
+      	                     <image>
+      	                        <Image url="@../content/information.png" />
+      	                     </image>
+      	                  </ImageView>
+      	               </graphic>
+      	               <HBox.margin>
+      	                  <Insets left="5.0" top="5.0" />
+      	               </HBox.margin>
+      	            </Label>
+              	</children>
+              </HBox>
+         </children>
+         <padding>
+            <Insets top="12.0" />
+         </padding>
+      </VBox> 
+    </children>
+    <padding>
+        <Insets bottom="10.0" left="10.0" right="10.0" />
+    </padding>
+</GridPane>
```

Comment:
@mikeknows, Do you have any idea what I could do to make it not appear that way? I forked the repository and then imported it through Eclipse. Since it's an XML-based file, I used FXEditor to make my changes to Activity.xml. Finally, I uploaded it to the same directory in my fork where Activity.xml lived. 

I did this with the other file, but it actually tracked my changes whereas this one didn't.

Reply:
Nevermind, I found the problem. It somehow got committed to the base directory instead of the view directory. It now shows additions and subtractions